public/js/matching-answer.js

```diff
@@ -0,0 +1,122 @@
+var matchAnswerCount = 3; // default number of options
+
+// Answering Variables
+var answeredId = 0;
+var answerList = [];
+var currentSelected = null;
+
+/**
+* Adds an option to the multiple choice
```

Comment:
adds an option to the matching? or multiple choice?

Reply:
oopsies will fix

Reply:
lol nw, i just tripped out for a sec 😜